tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/package-info.java

```diff
@@ -0,0 +1,227 @@
+package org.eclipse.n4js.ide.tests.server;
+
+/**
+ * Documentation defining default projects or workspaces in
+ * {@link org.eclipse.n4js.ide.tests.server.AbstractStructuredIdeTest}s and for their creation in
+ * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceCreator}.
+ */
```

Comment:
I think this javadoc should be put on the package declaration. Then it will be picked up by the JavaDoc generation tool as the package's documentation.

Also: add links to the sub-documentations below, then this servers as an entry point to the documentation.